rpc/enqueue.go

```diff
@@ -14,16 +14,11 @@ import (
 	"github.com/levenlabs/postmaster/sender"
 )
 
-// EnqueueArgs defines the arguments for the Postmaster.Enqueue endpoint
-type EnqueueArgs sender.Mail
+type enqueueArgs sender.Mail
```

Comment:
honestly we don't even really have to make an `enqueueArgs`, just use a `*sender.Mail` as args if you want